packages/api/resolvers/mutations/updateOrderDelivery.js

```diff
@@ -1,15 +1,35 @@
 import { log } from 'meteor/unchained:core-logger';
 import { OrderDeliveries } from 'meteor/unchained:core-orders';
-import { OrderDeliveryNotFoundError, InvalidIdError } from '../../errors';
+import { DeliveryProviderType } from 'meteor/unchained:core-delivery';
+import {
+  OrderDeliveryNotFoundError,
+  InvalidIdError,
+  OrderDeliveryTypeError,
+} from '../../errors';
+
+const DELIVERY_UPDATE_ENDPOINT = {
+  updateOrderDeliveryShipping: DeliveryProviderType.SHIPPING,
+  updateOrderDeliveryPickUp: DeliveryProviderType.PICKUP,
+};
 
 export default function updateOrderDelivery(
   root,
   { orderDeliveryId, ...context },
-  { userId }
+  { userId },
+  { fieldName }
```

Comment:
Good idea but I think we should solve that differently, having 2 explicit resolver functions instead of using fieldName and a custom map. Like this it's a little bit complex just for DRY's sake.